app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  @@renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
+
+  def render_markdown_as_html markup
+    raw @@renderer.render markup
```

Comment:
@cawel I see you're a fan of the Seattle style use of parentheses, but based on the style guide we've adopted:

> Omit parentheses around parameters for methods that are part of an internal DSL (e.g. Rake, Rails, RSpec), methods that have "keyword" status in Ruby (e.g. attr_reader, puts) and attribute access methods. Use parentheses around the arguments of all other method invocations.

https://github.com/bbatsov/ruby-style-guide#no-dsl-parens